description: function will not compile due to missing closing brace
author: Michael Gilbert <mgilbert@debian.org>

--- a/content/public/test/browser_test_base.cc
+++ b/content/public/test/browser_test_base.cc
@@ -653,6 +653,7 @@ void BrowserTestBase::ProxyRunTestOnMain
     base::ThreadRestrictions::SetIOAllowed(old_io_allowed_value);
     TearDownOnMainThread();
   }
+}
 
   if (base::CommandLine::ForCurrentProcess()->HasSwitch(
           switches::kEnableTracing)) {
